.circleci/config.yml

```diff
@@ -461,6 +472,7 @@ jobs:
       - run:
           name: Check pods and deployment targets
           command: bundle exec fastlane check_pods
+      - slack-notify
```

Comment:
This also runs on PRs, we don't need it here

Reply:
Hmm pod-lib-lint is executed also in `main` right? I'm planning to only remove the ones for jobs that are not supposed to run on main never